.husky/pre-commit

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-yarn lazy run build-api
+npx lazy run build-api
```

Comment:
is this to get around these not triggering correctly from within `bublic` due to a missing `YARN_RC_FILENAME`?

lu and i did an alternative version of this change where we checked if the containing folder was named bublic and manually set that var appropriately

Reply:
(this is fine too but if we wanted to stick with yarn for consistency then that works too!)

Reply:
https://github.com/tldraw/tldraw/pull/1215/files#diff-d2bc4bbf14eadc292d84e0ef5f7a93115c23557f533809fc80b896934291529dR4-R8

Reply:
> is this to get around these not triggering correctly from within bublic due to a missing YARN_RC_FILENAME?

Yeah exactly. npx does the right thing and shaves 30ms or so off the start time, so I'm inclined to stick with it.